<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LET UPX Encryption Tool (GUI)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:right;
border:1px solid rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Hamson@leee.tech</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkFast">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkNormal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkGood">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Good</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBetter">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Better</string>
    </property>
   </widget>
   <widget class="QLabel" name="compressTipsLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>461</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#helpBtn{
	border: 1px solid rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/help.png</normaloff>:/icon/help.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
